android-pdf-viewer/src/main/java/com/github/barteksc/pdfviewer/PDFView.java

```diff
@@ -1737,7 +1742,7 @@ public void refresh(int currPage) {
             PDFView.this.callbacks.setOnTapListener(onTapListener);
             PDFView.this.callbacks.setOnLongPressListener(onLongPressListener);
             PDFView.this.callbacks.setOnPageErrorListener(onPageErrorListener);
-            PDFView.this.callbacks.setLinkHandler(linkHandler);
+            PDFView.this.callbacks.setTapHandler(tapHandler);

```

Comment:
Continueing in the naming trend this should be renamed to `setOnTapListener` if you check the above naming pattern.

There is already a `onTap` listener so I'm wondering why we need to introduce here yet another another one, can you check and reuse that one instead?

Reply:
They use different kind of event, I don't see a way to combine them

![image](https://github.com/VERO-Digital-Solutions/AndroidPdfViewerV2/assets/57250076/0c0e2396-6295-435a-889e-e665b679d613)

Reply:
> They use different kind of event, I don't see a way to combine them

They do but you only make use of `x,y` coming from  a MotionEvent

![2024-02-13_09-12_2](https://github.com/VERO-Digital-Solutions/AndroidPdfViewerV2/assets/14945479/5c5a85ae-1ba1-4569-be3f-f0f942eaf513)
![2024-02-13_09-12_1](https://github.com/VERO-Digital-Solutions/AndroidPdfViewerV2/assets/14945479/63227719-cba8-4d1b-bbaa-bc9cd4b2fa75)
![2024-02-13_09-12](https://github.com/VERO-Digital-Solutions/AndroidPdfViewerV2/assets/14945479/b5639818-f988-4064-96e3-0df3ec1af8a1)